### DISCUSSION TITLE
How to use keystone embedded mode with astro.js?

### DISCUSSION THREAD

**Discussion by xspd3000** [keystonejs/keystone#7947] [Questions]
How can we use Keystone 6 with Astro 1.2?
How can we use Keystone 6 with React?

There is only one tutorial for using embedded mode.

**Comment by flexdinesh** [comment]
We have plans to add framework integration guides/examples. In the meanwhile you could browse around our [examples in the repo](https://github.com/keystonejs/keystone/tree/main/examples). 

[e2e](https://github.com/keystonejs/keystone/tree/main/examples/e2e-boilerplate) will give you an idea on how to use Keystone as a standalone API for a Next.js frontend. You can use that as an example to build other client rendered React or Astro frontend examples in the meanwhile.

**Reply by flexdinesh** [reply]
re: embedded mode

We're revising what embedded mode means. We're investigating how to embed Keystone into framework servers like Next.js/Astro. 🤞 we'll have something in this space soon.
